tests/unit/porting/npl/Makefile

```diff
@@ -0,0 +1,111 @@
+# Makefile
```

Comment:
I believe these are unit tests for Linux NPL so should be in `porting/npl/linux/tests`
if we need to have unit tests for NPL itself, there is already test framework which can be run via `newt` tool and it should go to `porting/test`